app/views/works/index.html.erb

```diff
@@ -0,0 +1,91 @@
+<html>
+<main>
+<h2>List of Works</h2>
+ <section class="media-table">
+   <h4>Album</h4>
+   <table class="table">
+     <thead>
+       <tr>
+         <th width=80>Votes</th>
+         <th>Title</th>
+         <th>Created By</th>
+         <th width=100>Published</th>
+         <th width=100>Upvote</th>
+       </tr>
+     </thead>
+     <% @albums.each do |album| %>
+     <tbody>
+         <tr>
+           <td><%= album.votes.count %></td>
+           <td><%= link_to album.title, work_path(album) %></td>
+           <td><%= album.creator %></td>
+           <td><%= album.publication_year %></td>
+           <td><a class="btn btn-primary" <%= link_to "Upvote", new_vote_path(album.id), method: :post %></a>
+             </td>
+         </tr>
+     </tbody>
+     <% end %>
+   </table>
+ </section>
+
+ <section class="media-table">
+   <h4>Book</h4>
+   <table class="table">
+     <thead>
+       <tr>
+         <th width=80>Votes</th>
+         <th>Title</th>
+         <th>Created By</th>
+         <th width=100>Published</th>
+         <th width=100>Upvote</th>
+       </tr>
+     </thead>
+     <% @books.each do |book| %>
+     <tbody>
+         <tr>
+           <td><%= book.votes.count %></td>
+           <td><%= link_to book.title, work_path(book) %></td>
+           <td><%= book.creator %></td>
+           <td><%= book.publication_year %></td>
+           <td><a class="btn btn-primary" <%= link_to "Upvote", new_vote_path(book.id), method: :post %></a>
+             </td>
+         </tr>
+     </tbody>
+     <% end %>
+   </table>
+ </section>
+
+ <section class="media-table">
+   <h4>Movie</h4>
+   <table class="table">
```

Comment:
Notice how these tables are repudiative, you could DRY them out with a partial view.